packages/machine/src/protocol/install-virtual-app.ts

```diff
@@ -270,8 +272,10 @@ function createAndAddTarget(
     appInterface,
     {
       assetType: AssetType.ETH,
-      limit: Zero, // limit field is ignored, since limits are enforced by virtual app agreement
-      token: AddressZero
+      limit: bigNumberify(initiatingBalanceDecrement).add(
+        bigNumberify(respondingBalanceDecrement)
```

Comment:
add a comment: the limit field is used for resolve, but not respected by `transfer`, instead the virtual app agreement enforces its own limit